tests/ops/api/v1/endpoints/test_connection_config_endpoints.py

```diff
@@ -767,6 +767,74 @@ def test_search_connections(
         assert ordered[0].key == items[0]["key"]
         assert ordered[1].key == items[1]["key"]
 
+    @pytest.mark.unit_saas
```

Comment:
yeah i wasn't exactly sure why we have the `unit_saas` marker, but afaik it _does_ get included in the default unit test runs? i just included it to indicate that there are saas artifacts used, and some sass-related functionality tested, in the test. anyway, i'm totally fine with removing it - wasn't really convinced i should include it in anyway

Reply:
I think you are right about it being included in the default unit tests since those exclude marks rather than include them. I don't  have a strong opinion that it has to be removed so if it makes sense to keep it that is fine.